rfc/cart-api-v1.md

```diff
@@ -0,0 +1,179 @@
+# Version 1
+
+| Issue | Status |
+| ----- | ------ |
+| #88   | Draft  |
+
+## Data Models
+
+There are two data models required for Cart API: `Cart` and `CartItem`. They are
+designed in the way that allows multiple types of entity in a single `Cart`.
+
+### `Cart`
+| Property | Type | Description | Note |
+| --- | --- | --- | --- |
+| `id` | UUID | Cart ID, auto-generated by the web API | - |
+| `user_id` | UUID | The ID of the user that creates the collection | - |
+| `cart_name` | `str` | Cart Name: If it is `NULL`, the cart is the default one for a corresponding user. | **NEED MORE DISCUSSION** |
+| `default_cart` | `bool` | Flag to indicate whether this cart is the default one for a corresponding user (`False` by default) | **NEED MORE DISCUSSION** |
+
+For the details on the default cart, see `GET /resources/carts/{id}`.
+
+### `CartItem`
```

Comment:
I think we should also store some user-readable data about the item so the user can distinguish items in the cart.
Storing all the metadata for each entity type probably isn't feasible so something generic across the entity types may be good enough.  e.g. `ItemName`